### PR TITLE
Add Cloud team to handbook and move MI docs from DevOps page to the Cloud team page

### PR DESCRIPTION
**Pull request by RafLeszczynski**

### CODE REVIEW

File: content/departments/cloud/index.md [hunk 1]
```diff
@@ -0,0 +1,185 @@
...
+- Compatible with current MI use cases
+  - Infrastructure / Domain / Isolation boundary per customer
+
+**Dedicated Sourcegraph instances**
+
+- One Sourcegraph instance serves a single customer
+- (FY23/FY24) Dedicated Cloud infrastructure
+  - With a path to “single-tenant Sourcegraph, shared-infrastructure” model in the future
+- (FY23/FY24) GCP only
+  - With a path to supporting other Cloud providers / Bring-your-own-Kubernetes in the future
```
Comment (filiphaftek) [inline content/departments/cloud/index.md]: `Bring-your-own-Kubernetes` - eager to see this :) b/c this sounds very hard to implement and maintain for us - I would be more for`customize your K8s` on available providers THB.

**Review by filiphaftek — APPROVED** [review]
Nice!

**Comment by daxmc99** [comment]
@RafLeszczynski Added a few additional changes here we can merge in now or after: https://github.com/sourcegraph/handbook/pull/4207

File: content/departments/cloud/index.md [hunk 2]
```diff
@@ -0,0 +1,185 @@
+# Cloud
+
+> NOTE: **Cloud means single-tenant dedicated instances managed by Sourcegraph** _(for example `mycompany.sourcegraph.com`)_. Sourcegraph Cloud should not be confused with Sourcegraph.com which holds public and open source code. The Cloud and managed instance should be considered synonyms within these handbook pages.
+
+The Cloud team is the special focus team reporting directly to CEO modeled on _“if AWS were to offer ‘Managed Sourcegraph’ like they do Elasticsearch, Redis, PostgreSQL, etc., how would they do it?”_ The team is responsible for maintaining existing [managed instances](https://docs.sourcegraph.com/admin/install/managed). and building the next generation of them. The Cloud team has no other responsibilities.
+
+## Mission statement
+
+Build a **fully managed platform** for using Sourcegraph that can **(by EOFY23) support 100+ customers** using **dedicated** Sourcegraph instances, providing **feature compatibility** with self-hosted while **being cost-efficient for customers and Sourcegraph**.
```
Comment (rafax) [inline content/departments/cloud/index.md]:
```suggestion
Build a **fully managed platform** for using Sourcegraph that can **(by EOFY23) support 200+ customers** using **dedicated** Sourcegraph instances, providing **feature compatibility** with self-hosted while **being cost-efficient for customers and Sourcegraph**.
```

File: content/departments/cloud/index.md [hunk 3]
```diff
@@ -0,0 +1,185 @@
...
+
+**Platform**
+
+- Low customer onboarding cost
+- Zero customer maintenance cost
+- Secure (SOC 2, documented security posture)
+- Reliable (ability to offer SLA, internal SLO of 99.9%)
+- Automatable (in due time, feature releases / billing / upgrades / analytics are built-in)
+
+**Support 100+ customer**
```
Comment (rafax) [inline content/departments/cloud/index.md]:
```suggestion
**Support 200+ customer**
```
Updated following our discussion with @sqs (I realize original doc had 100)

File: content/departments/cloud/index.md [hunk 4]
```diff
@@ -0,0 +1,185 @@
...
+
+- Targeting 100+ customers in FY23 to invest in supporting 1000 in FY24
```
Comment (rafax) [inline content/departments/cloud/index.md]:
```suggestion
- Targeting 200+ customers in FY23 to invest in supporting 1000 in FY24
```

File: content/departments/cloud/index.md [hunk 5]
```diff
@@ -0,0 +1,185 @@
...
+  - Support 250 production-grade instances (accommodating trials / testing)
```
Comment (rafax) [inline content/departments/cloud/index.md]:
```suggestion
  - Support 300 production-grade instances (accommodating trials / testing)
```

File: content/departments/cloud/index.md [hunk 6]
```diff
@@ -0,0 +1,185 @@
...
+- Compatible with current MI use cases
+  - Infrastructure / Domain / Isolation boundary per customer
+
+**Dedicated Sourcegraph instances**
+
+- One Sourcegraph instance serves a single customer
+- (FY23/FY24) Dedicated Cloud infrastructure
```
Comment (rafax) [inline content/departments/cloud/index.md]:
```suggestion
- (FY23/FY24) Dedicated, Sourcegraph-provided Cloud infrastructure
```

File: content/departments/cloud/index.md [hunk 7]
```diff
@@ -0,0 +1,185 @@
...
+  - With a path to “single-tenant Sourcegraph, shared-infrastructure” model in the future
```
Comment (rafax) [inline content/departments/cloud/index.md]: Given our current plans, I'd remove this

File: content/departments/cloud/index.md [hunk 8]
```diff
@@ -0,0 +1,185 @@
...
+- (FY23/FY24) GCP only
+  - With a path to supporting other Cloud providers / Bring-your-own-Kubernetes in the future
```
Comment (rafax) [inline content/departments/cloud/index.md]: Given our current plans, I'd remove this

Comment (rafax) [inline content/departments/cloud/index.md]:
```suggestion

```

File: content/departments/cloud/index.md (hunk 7 above)
Comment (rafax) [reply content/departments/cloud/index.md]:
```suggestion
```

File: content/departments/cloud/index.md [hunk 9]
```diff
@@ -0,0 +1,185 @@
...
+  - With time, getting more powerful than self-hosted
+- Features are opt-in (for a fee)
+- New features available on Cloud before self-hosted
+- Existing features have higher adoption on Cloud than self-hosted
+
+**Cost-efficiency**
+
+- Expected to support teams from 50 to 5000 users (EOFY23) at 500$/month minimal infrastructure cost
+- Infrastructure cost covered by Sourcegraph
+  - We’re incentivized to operate efficiently
```
Comment (rafax) [inline content/departments/cloud/index.md]:
```suggestion
```

File: content/departments/cloud/index.md [hunk 10]
```diff
@@ -0,0 +1,185 @@
...
+- Administration / operations provided by Sourcegraph
+  - Cloud team headcount scales sublinearly with number of customers
```
Comment (rafax) [inline content/departments/cloud/index.md]:
```suggestion
```

File: content/departments/cloud/index.md [hunk 11]
```diff
@@ -0,0 +1,185 @@
...
+- (FY24) Self service provisioning / release channels for upgrades
+
```
Comment (rafax) [inline content/departments/cloud/index.md:53]:
```suggestion

Not in scope (for FY23 / FY24):

- supporting customer provided GCP infrastructure
- supporting cloud providers other than GCP
- managing Sourcegraph installations in clusters not provisioned by the Cloud team (Bring-your-own-Kubernetes)
- supporting customers smaller than [X1](https://docs.google.com/document/d/1jJq_-arjzjsBDFWV5VlHi01KrxTXYd8NV5UV-fmMtCE/edit#bookmark=id.kjizf2bpwb0f) ARR
- optimizing cost below [X2](https://docs.google.com/document/d/1jJq_-arjzjsBDFWV5VlHi01KrxTXYd8NV5UV-fmMtCE/edit#bookmark=id.pqt5n89wdcan) $/month

```

File: content/departments/cloud/index.md [hunk 12]
```diff
@@ -0,0 +1,185 @@
...
+### Q2FY23 goals
+
+- Support up to 100 managed instances [without greatly increasing engineering capacity invested in maintenance](https://docs.google.com/document/d/1SvWVScwBtLdrvoVKB4tgSiqhv3MPBDjv6JbwqGnlLdg/edit#)
+  - [Migrate all managed instances to v 1.1 architecture](https://docs.google.com/document/d/1ZHOnZc6K0oLhnqbcFYyzvdqng2ZM4vhc8Ln20Q4piQ0/edit#heading=h.trqab8y0kufp)
+- Provision sourcegraph.sourcegraph.com as production grade managed instance for Sourcegraph team
+- [Support migration Sourcegraph.com customers to the Cloud](https://docs.google.com/document/d/1owIo8QA_omwnFqSfS5EKyIhd4SpMDdn7cZJrsvgjl-E/edit#heading=h.2vev9l6i9qrg)
+- Support SOC2 audit for managed instances
+- Enabling managed instances trials (limited capacity)
+- Defined SLO & measured SLO for managed instances
```
Comment (rafax) [inline content/departments/cloud/index.md]:
```suggestion
- Defined SLO & measured SLO for managed instances
- Finalize design & implementation plan of [RFC 706](https://docs.google.com/document/d/1pPS3xzv4pI5ZlliPrr1tkoOtvn-ZWxKVdYfNNHKVPHs/edit)
```

**Review by rafax — APPROVED** [review]

**Review by michaellzc — APPROVED** [review]
lgtm

**Pull request merged**